### DISCUSSION TITLE
Issue related to an Article based on AlphasimR

### DISCUSSION THREAD

**Discussion by Dkplucas** [gaynorr/AlphaSimR#135] [Q&A] [answered]

**Comment by gregorgorjanc** [comment]
@Dkplucas this is a work in progress so I would not suggest to use this code. May I ask where did you get the script from?

**Reply by Dkplucas** [reply]
Sorry I did not know the work was stillin progress. I was training my coding skill in AlphasimR. I will delete the script and wait for you to finish.

**Comment by gregorgorjanc** [comment, marked as answer]
@Dkplucas I see now that this is your own code. We have a working code for such a dairy cattle breeding programme simulation that we have successfully used on our end. We will make this code available soon so that people can see how we managed memory - there are some important points to enable large-scale simulations such as we describe in that publication with ~500K animals active in each year. Stay tuned;)

**Comment by Dkplucas** [comment]
Sure, I am looking forward to hearing from you. In meantime, I am working on African cattle breed genetic improvement by simulation. I have already written the script for the breeding program. I am just waiting to improve the estimation of the breeding value for the parent. Since I have learned everything from you, I would like to cite you as an author for the publication. And will send you the script for correction before publication. If you would mind and accept my request. Please if possible can I have your mail address so I can directly send you the working material to avoid exposing it online. Thanks in advance for your help and looking forward to learn more from you Professor.

**Comment by gregorgorjanc** [comment]
@Dkplucas this is kind of you and much appreciated! Just make sure you cite the AlphaSimR paper ;)

https://doi.org/10.1093/g3journal/jkaa017

**Comment by Dkplucas** [comment]
Sure Professor, Thanks for your help.
